zipline/testing/fixtures.py

```diff
@@ -384,10 +445,38 @@ class WithBcolzDailyBarReader(WithTmpDir, WithSimParams):
     - `cls.bcolz_daily_bar_reader` is a daily bar reader pointing to the
       directory that was just written to.
 
+    Attributes
+    ----------
+    BCOLZ_DAILY_BAR_PATH : str
+        The path inside the tmpdir where this will be written.
+    BCOLZ_DAILY_BAR_LOOKBACK_DAYS : int
+        The number of days of data to add before the first day in the
+        sim_params. This is used when a test needs to use history, in which
+        case this should be set to the largest history window that will be
+        requested.
+    BCOLZ_DAILY_BAR_FROM_CSVS : bool
```

Comment:
This feels like it just belongs in a subclass that overrides `make_daily_bar_data`.